docs/conf.py

```diff
@@ -47,21 +48,25 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Trytond UPS Integration'
-copyright = u'2014, Openlabs Technologies & Pvt. Ltd.'
+project = u'Tryton UPS Shipping Integration'
+copyright = u'2015, Openlabs'
+author = u'Openlabs'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = '1.0'
+version = 'VERSION'
```

Comment:
I think the version and release should get the information from environment. See example here: 

https://github.com/openlabs/nereid/blob/develop/docs/conf.py#L56